required/tools/array.dtx

```diff
@@ -2486,11 +2766,41 @@
 %    =\@array=. To save \textsf{token} space we then redefine
 %    =\@preamble=
 %    because its replacement text isn't longer needed.
+%
```

Comment:
The $ around 20 lines above should be surrounded by something like `\FakeMathStart$\FakeMathStop `   to avoid that it triggers math tagging. The commands can be defined as dummies (in lttagging.dtx?) and the math code can then redefine them.

Reply:
We have to talk about that suggested  fakemath interface, I think it is wrong, but that isn't for this pull.

Reply:
It would be really good if there were directly an interface (can be internal)  to suppress the math grabbing here. Otherwise it is hard to tag math inside tabulars (one has to disable it outside the tabular and reenable it in every cell) and I need that for the larger math example ... (also it would resolve an open issue with tabularx).

Reply:
Again, I can't put anything in that is only in your private version of tagpdf, because if that then goes out as latex-dev it will blow up. Anway, in my opinion the interface should  just say `\FakeMathStart` hiding the `$` inside (and whatever is necessary to ensure that inner math is still grabbed and tagged --- which may be nothing if the command is protected)

Reply:
Sorry? There is nothing about this in my private tagpdf. We simply need here something that allows us to say "this $ should not do math grabbing" and does not affect following math.

Reply:
sorry not in your tagpdf but in your branch for math coding which is not released

Reply:
but why should it blow up? I only asked that there is something  around the dollar which does nothing by default but can be redefined by the math code to suppress the math grabbing in this place.